.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false    #container 
+
+language: python
+python:
+  - "2.7"
+  - "3.6"
+  # - "3.7"  fails build 'Unable to download 3.7 archive. The archive may not exist.''
+
+notifications:
+  email: never
+
+install:
+  - pip install -r requirements.txt
+  - pip install codecov
```

Comment:
Maybe combine all these pip installs into one line?

Reply:
or should i put all inside requirments.txt? and have two different requiremnts - one for development and one for the actual package?